proto/storage/node.proto

```diff
@@ -86,11 +89,28 @@ message NodeScan {
 }
 
 message NodeInventory {
-    string                         node_id    = 1 [(gogoproto.moretags) = 'search:"Node ID,store" sql:"pk,type(uuid)"'];
-    string                         node_name  = 2 [(gogoproto.moretags) = 'search:"Node,store"'];
-    google.protobuf.Timestamp      scan_time  = 3 [(gogoproto.moretags) = 'search:"Node Scan Time,store"'];
-    scannerV1.Components           components = 4;
-    repeated scannerV1.Note        notes      = 5;
+    string                         node_id    = 1;
```

Comment:
`NodeInventory` is a field within `Node` which has `id` and `name`. Why are we adding id and name again?

Reply:
`NodeInventory` is generated in Compliance and has only node name at that stage. Once it arrives to Sensor, we compare node name against all other node names known to Sensor to connect these two by attaching `NodeInventory` to `Node`. 

You are right, node ID is not required for `NodeInventory` anymore. I will remove it here.

Reply:
We were thinking about separating `NodeInventory` from `Node`, but this would cause a lot of extra work. Instead, we think that `NodeInventory` does not make a lot of sense if it is detached from the `Node`.

Moreover, the information about `NodeInventory` will be duplicated in the DB - the same information will be available in the `NodeScan` field that is returned from Scanner (but it will include all information about the vulnerabilities).

The flow of messages goes like this: `Sensor -> Node+NodeInventory -> Central Pipeline -> NodeInventory + 4 fields of Node -> Scanner -> NodeScan -> Central -> Node+NodeScan -> DB`. 
I think there is no need to store  `Node+NodeInventory+NodeScan` as the information would be duplicated. What do you think?

Alternatively, we may add `NodeInventory` to the `Node` in the DB, but I would need some support for this from DB experts. How this would need to be done if we would like to go that way?

Reply:
@md2119 @keyallis could I ask you to give this PR another look?

We still keep the proto changes as last time, but the `NodeInventory` is now present in the Database in a serialized form. 
I was wrong last time saying that it is not stored in the DB - what I meant was that there will be no extra tables or fields created in the DB for NodeInventory. I corrected the PR description to be more precise.

This PR blocks a major part of the work on RHCOS epic and I would love to move forward with it.